zbus/src/connection/builder.rs

```diff
@@ -89,7 +89,10 @@ impl<'a> Builder<'a> {
         Self::address(&crate::address::system()?)
     }
 
-    /// Create a builder for an IBus connection
+    /// Create a builder for the [IBus] daemon.
+    ///
+    /// [IBus]: https://en.wikipedia.org/wiki/Intelligent_Input_Bus
```

Comment:
this is a fix to your first commit here and should be part of that, not the one introducing the feature. Of course, if you go for squashing them both, then it'll be fixed by that automatically. :)

Reply:
Oh, should I fix the first commit as a new one? Introducing the feature be another one commit?

Reply:
> Oh, should I put the fix of first commit as a new one?

No, the fixes to the (merged) commits should be squashed into the commits they're fixing.

> Introducing the feature be another one commit?

Ideally, yes. This specific change is not the one introducing the cargo feature so this (only talking about these lines here) should be part of the commit that adds the new method. If you keep the introduction of the cargo feature are a separate commit, that commit should come first obviously because the method uses it just before this line here. You'll have to split the commit and then move part of it before.

Alternatively, you can just keep it simple and have both changes in one commit but then please add a bit more description to the commit log and mention that fact that it's optional and not enabled by default.